aleph-client/src/pallets/staking.rs

```diff
@@ -424,13 +419,14 @@ impl<C: AsConnection + Sync> StakingRawApi for C {
     ) -> anyhow::Result<Vec<StorageKey>> {
         let key_addrs = api::storage().staking().eras_stakers_root();
         let mut key = key_addrs.to_root_bytes();
-        StorageMapKey::new(era, StorageHasher::Twox64Concat).to_bytes(&mut key);
-        self.as_connection()
-            .as_client()
-            .storage()
-            .fetch_keys(&key, 10, None, at)
-            .await
-            .map_err(|e| e.into())
+        key.extend(subxt::ext::sp_core::twox_64(&era.encode()));
```

Comment:
Perhaps ugly, but this is the way:  https://github.com/paritytech/subxt/blob/43809ee5928a77eed5438f7d61f24fe0183f970a/testing/integration-tests/src/full_client/storage/mod.rs#L78.